### ISSUE TITLE
Waking for undetermined reason

### ISSUE THREAD

**Issue by lividhen** [scramble45/decky-wol#1]
~~This is super useful, but something keeps waking up my deck randomly 😅. Is there any way to figure out what it would be? Does the plugin log packets and / or where they're sent from?~~

After letting Wireshark run for a few days, there are no wol packets sent over the network. I can not seem to determine the cause of waking.

**Comment by scramble45** [comment]
yeah I am not 100% sure the root cause of that, I need some more time to investigate. Thanks for opening an issue, if you determine the root cause lemme know and I will take a look or feel free to submit a PR.

**Comment by lividhen** [comment]
Cool, thanks! I have found that most distros are terrible at giving a wake up reason which is making it hard to debug. There is probably a log or a software somewhere that can help but I haven't found it yet.

**Comment by scramble45** [comment]
> Cool, thanks! I have found that most distros are terrible at giving a wake up reason which is making it hard to debug. There is probably a log or a software somewhere that can help but I haven't found it yet. 

If you look two systemd service are created. Try disabling the one which unblocks hardware. And see if that has any effect, the one that interrupts the hibernation process will have to stay active so hopefully that's not it. But perhaps the unblock isn't needed, still need to test that further. It could be some other timer that is triggered outside of pressing the sleep button on the deck that causes this odd behavior with the other service that does the hibernation redirect to sleep.
